src/main/scala/akka/persistence/cassandra/testkit/CassandraLauncher.scala

```diff
@@ -89,9 +93,26 @@ object CassandraLauncher {
    * @throws akka.persistence.cassandra.testkit.CassandraLauncher.CleanFailedException if `clean`
    *   is `true` and removal of the directory fails
    */
-  def start(cassandraDirectory: File, configResource: String, clean: Boolean, port: Int): Unit = this.synchronized {
-    if (cassandraDaemon.isEmpty) {
+  def start(cassandraDirectory: File, configResource: String, clean: Boolean, port: Int): Unit =
+    start(cassandraDirectory, configResource, clean, port, fork = false)
```

Comment:
It works in sbt because we have to fork the tests. If this solves it then it's a much better solution, and would be a better default. A good test is to run the tests in this project without sbt forking and with launcher forking.

I don't mind changing the default for this, also without deprecation phase. It's just a test utility.